src/site/artifacts/page.rs

```diff
@@ -42,11 +41,6 @@ pub fn build(config: &Config) -> Result<()> {
     }) = &config.artifacts
     {
         html.extend(cargo_dist::build_table(manifest, config));
-    };
-
-    let doc = layout::build(config, html!(<div>{html}</div>), false)?;
-    let html_path = format!("{}/artifacts.html", &config.dist_dir);
-    let asset = axoasset::local::LocalAsset::new(&html_path, doc.into());
-    axoasset::local::LocalAsset::write(&asset, &config.dist_dir)?;
-    Ok(())
+    }
+    Ok(html!(<div>{html}</div>).to_string())
```

Comment:
@SaraVieira should i be worried about adding this extra div here? it was the only thing i could do to get this going...

Reply:
Unless it breaks the layout it's all good

Reply:
ok it doesn't seem to